config/webhook/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 resources:
 - manifests.yaml
-#- service.yaml
+- service.yaml
```

Comment:
We might want to comment this again, as we don't actually use the webhook service